paypal/standard/ipn/views.py

```diff
@@ -31,6 +35,11 @@ def ipn(request):
     flag = None
     ipn_obj = None
 
+    # Avoid the RawPostDataException. See original issue for details:
+    # https://github.com/spookylukey/django-paypal/issues/79
+    assert request.META.get('CONTENT_TYPE', '').startswith(
+        'application/x-www-form-urlencoded'), CONTENT_TYPE_ERROR
+
```

Comment:
In theory, 'asserts' can be optimized away (e.g. if you run Python with -O flag).

So this should be changed to an explicit raise. It should probably be an `AssertionError`, then your tests won't have to change.